roles/journald/handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+# handlers file for journald
+
+- name: restart journald
+  command: systemctl restart systemd-journald
```

Comment:
Could you move it into service:
- name: restart journald
  service:
    name: journald
    state: restarted

Reply:
done